main/windows/index.ts

```diff
@@ -206,19 +175,10 @@ class Tray {
         const gasDisplay = Math.round(hexToInt(gasPrice) / 1000000000).toString()
         title = gasDisplay // ɢ 🄶 Ⓖ ᴳᵂᴱᴵ
       }
-      this.electronTray.setTitle(title)
+      setSystemTrayTitle(title)
     })
     this.readyHandler = () => {
-      this.electronTray.on('click', () => {
-        this.recentElectronTrayClick = true
-        clearTimeout(this.recentElectronTrayClickTimeout as NodeJS.Timeout)
-        this.recentElectronTrayClickTimeout = setTimeout(() => {
-          this.recentElectronTrayClick = false
-        }, 50)
-        if (process.platform === 'win32') {
-          this.toggle()
-        }
-      })
+      initSystemTray(this.toggle)
```

Comment:
we already have a class that maintains a reference to the tray (this one), I don't think we need another module that holds a reference to it that doesn't really add any benefit. the context menu module doesn't need a reference to the tray, we can just invoke it with an API like:

```
this.electronTray.setContextMenu(createContextMenu('show', onClick = () => {}, displaySummonShortcut))
``

Reply:
Where's the reference to the system tray in this class?

Reply:
you moved it into the new module

Reply:
So it's no longer maintaining a reference, it's in the new module.

Reply:
I'm reworking it again, but just FYI the Electron system tray can't be created before the app is ready.